data-solutions/data-platform-foundations/01-environment/variables.tf

```diff
@@ -55,3 +55,17 @@ variable "service_account_names" {
     main = "data-platform-main"
   }
 }
+
+variable "service_encryption_key_ids" {
+  description = "Cloud KMS encryption key in {LOCATION => [KEY_URL]} format. Keys belong to existing project."
+  type = object({
+    regional      = string
+    multiregional = string
+    global        = string
+  })
+  default = {
+    regional      = null
+    multiregional = null
```

Comment:
the code is using only multiregional.
1. Do we need the other options?
2. If yes (I assume we need it), "how to use" example can be nice.

Reply:
Regional key not in use at the moment, removed.